notebook/db_util.py

```diff
@@ -1,171 +1,125 @@
+        :param kernel_id: (str) UUID of the KernelSession which is to be deleted
         """
-        kwargs = {column_name: column_value}
-        return KernelSession.objects.filter(**kwargs).delete()
+        requests.delete(urljoin(KERNEL_SESSION_API, kernel_id), auth=HTTPBasicAuth(USERNAME, PASSWORD))
 
-    def get_kernel_session(self, column_name, column_value):
+    def get_kernel_session(self, kernel_id: str) -> dict:
         """
-        Get kernel session instances for the given column name and value.
-        :param column_name: (str) Name of the column
-        :param column_value: (str) Value of column
-        :return: List of KernelSession instances.
+        Get the first instance of kernel session which matches the given column name and value.
+        :param kernel_id: (str) UUID of the KernelSession which is to be deleted
+        :return: A single JSON serialized KernelSession instance
         """
-        kwargs = {column_name: column_value}
-        return KernelSession.objects.filter(**kwargs)
+        resp = requests.get(urljoin(KERNEL_SESSION_API, kernel_id), auth=HTTPBasicAuth(USERNAME, PASSWORD))
+        return resp.json()
 
-    def create_kernel_session(self, _field_values):
+    def create_kernel_session(self, _field_values: dict) -> dict:
         """
         Create kernel session instance with provided values.
         :param _field_values: (Dict) Dictionary of field values.
-        :return: KernelSession instance.
+        :return: JSON serialized KernelSession instance.
         """
-
-        _kernel_id = _field_values.get("kernel_id", None)
-        _kernel_name = _field_values.get("kernel_name", None)
+        _kernel_id = _field_values.get("kernel_id")
+        _kernel_name = _field_values.get("kernel_name")
         mlnode_id = _field_values['mlnode_id']
-        result, _ = KernelSession.objects.update_or_create(kernel_id=_kernel_id,
-                                                           kernel_name=_kernel_name,
-                                                           ml_node=EdgeDevice.objects.get(id=mlnode_id))
-        return result
+        data={
+            "kernel_id": _kernel_id,
+            "kernel_name": _kernel_name,
+            "ml_node": int(mlnode_id),
+        }
+        resp = requests.post(KERNEL_SESSION_API, data=data, auth=HTTPBasicAuth(USERNAME, PASSWORD))
+        return resp.json()
 
-    def check_kernel_sessions(self, column_name, column_value):
+    def check_kernel_sessions(self, kernel_id: str) -> bool:
         """
         Check if there is an entry wih the following field and value.
-        :param column_name: Name of the Column. ( String )
-        :param column_value: Value of the Column. ( String )
+        :param kernel_id: UUID of the Kernel Session ( String )
         :return: Boolean Value.
         """
+        kernel_session = self.get_kernel_session(kernel_id)
+        return 'kernel_id' in kernel_session 
```

Comment:
is `kernel_id` supposed to be string?

Reply:
Just checking if in the JSON response we have got the field, can be handled better I will do these changes in a separate PR